get_stats.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+require_once('../../config.php');
+require_once(__DIR__ . '/lib.php');
+require_once($CFG->libdir . '/badgeslib.php');
+
```

Comment:
I'd add a require_login check here
Also, you can define something as being used for AJAX with a constant definition at the top of the file - e.g:

```
<?php
define('AJAX_SCRIPT', true);